contrib/datawave-quickstart/bin/services/accumulo/bootstrap.sh

```diff
@@ -15,7 +15,10 @@ DW_ACCUMULO_SERVICE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # Zookeeper config
 
 # You may override DW_ZOOKEEPER_DIST_URI in your env ahead of time, and set as file:///path/to/file.tar.gz for local tarball, if needed
+# DW_ZOOKEEPER_DIST_URI should, if possible, be using https. There are potential security risks by using http.
 DW_ZOOKEEPER_DIST_URI="${DW_ZOOKEEPER_DIST_URI:-https://archive.apache.org/dist/zookeeper/zookeeper-3.7.1/apache-zookeeper-3.7.1-bin.tar.gz}"
+# The sha512 checksum for the tarball. Value should be the hash value only and does not include the file name. Cannot be left blank.
+DW_ZOOKEEPER_DIST_SHA512_CHECKSUM="9103628a50745fa1a289bca666fda4a9c08ec17c55cf13e66887e7ba76e93dbae60a1f1ffd6c10798be3a16069344ecbc00cebb29bf03d9cd7096ccd098ed011"
```

Comment:
Is there not a separate link to pull down the checksum?

Reply:
Yes. There are separate links for most, if not all, the service dist URIs in the quickstart. My thought behind not using them was that if users decide to use other distribution URIs, those might not have specific links to pull the checksums. I can change the logic here and make it use either a link or a hash. Maybe my reasoning is wrong... but I am open to more feedback.

Reply:
Whether separate link or not, there needs to be a way to override the value in the user's environment, as with the `*DIST_URI` values, so that the user doesn't have to modify the source code here to experiment with different versions. E.g....
```
DW_ZOOKEEPER_DIST_SHA512_CHECKSUM=${DW_ZOOKEEPER_DIST_SHA512_CHECKSUM:-9103628a50745fa1a289bca666fda4a9c08ec17c55cf13e66887e7ba76e93dbae60a1f1ffd6c10798be3a16069344ecbc00cebb29bf03d9cd7096ccd098ed011}
```
With that said, it's probably more straightforward just to leave as checksums, rather than links to checksums, so that the user can easily build/install their own local version of a tarball, if desired. That's a fairly common use case, as when testing Accumulo SNAPSHOT versions, for example

Reply:
Good point. I went ahead and made changes based on your feedback.